web/style.css

```diff
@@ -226,6 +227,7 @@ input[type="range"]::-ms-fill-upper {
   font-weight: 800;
   padding: 0.5rem 1rem;
   color: var(--text);
+  width: 5rem;
```

Comment:
i think `5rem` is not wide enough, as right now the "pause" label is overflowing by a little bit:

<img width="494" alt="Screenshot 2024-07-23 at 23 20 04" src="https://github.com/user-attachments/assets/cd54bfa2-6b22-4736-b881-eaf4bbdc97d7">

as you can see, the label is not perfectly centered. i think a longer button should mitigate this issue.

Reply:
also the screenshot above shows that the backdrop of the button is 1px off to the right. for comparison, this is how it looks like in prod:

<img width="481" alt="Screenshot 2024-07-23 at 23 23 41" src="https://github.com/user-attachments/assets/1abe395c-b654-4083-8037-8e9a05af881d">